### DISCUSSION TITLE
How to clenup table event_logs?

### DISCUSSION THREAD

**Discussion by VA-PLE** [dagster-io/dagster#12985] [Q&A] [answered]
Hi. I noticed that the dagster database takes up a lot of disk space. The largest table is event_logs (99% size)

| event_logs         | 18 GB      |

How can I clear these logs? Can I set a retention period for these logs?

**Comment by tacastillo** [comment, marked as answer]
https://github.com/dagster-io/dagster/discussions/12047

Hey Vasyl! Have you seen this post on how to programmatically clean up the event logs? The summary is that you can use `DagsterInstance.get()`, fetch the run logs from there, and then delete those runs and their logs.

I don't believe there is a way to set a retention policy, but I'll take that feedback back to the team!

If that satisfies your needs, I'll mark this discussion as a duplicate of that linked one!

**Reply by VA-PLE** [reply]
Thank you for your help. Yes, I saw this message. I hope the feature "set a retention period for logs" will be added in the near future)

**Reply by luqmansen** [reply]
+1 on retention period for logs

**Reply by abuchmueller** [reply]
+1 on log retention

**Reply by tomas-mazak** [reply]
+1 on log retention

**Reply by menzenski** [reply]
+1 on configurable log retention

**Reply by Last-Uchiha18** [reply]
+1 on configurable log retention

**Reply by mar0-qed** [reply]
+1 on configurable log retention

**Comment by hnykda** [comment]
There is now this? https://docs.dagster.io/deployment/oss/oss-instance-configuration#data-retention

**Reply by Last-Uchiha18** [reply]
I believe it works only for sensor/schedule ticks. Not the actual runs stored in the event logs
